src/KeyedData/KeyedDataArray.php

```diff
@@ -97,7 +97,7 @@ public function search(string $path): mixed
         return JmesPath::search($path, $this->value);
     }
 
-    private function expand(string $path): array
+    public function expand(string $path): array
```

Comment:
Why?

Reply:
https://github.com/DrechslerDerek/data/blob/e12243ec9cfd4b551849c3ce206f9d1bf3bab4ad/src/JsonDataMutable.php#L52
needed here

Reply:
Then I think it should be made into a trait.